packages/sirius-web/backend/sirius-web-graphql/src/main/java/org/eclipse/sirius/web/graphql/datafetchers/representation/RepresentationMetadataDescriptionDataFetcher.java

```diff
@@ -42,6 +48,8 @@
 @QueryDataFetcher(type = "RepresentationMetadata", field = "description")
 public class RepresentationMetadataDescriptionDataFetcher implements IDataFetcherWithFieldCoordinates<CompletableFuture<IRepresentationDescription>> {
 
+    static final BiFunction<VariableManager, String, IStatus> FAKE_RENAME_HANDLER = (a, b) -> new Failure("");
```

Comment:
Alway specify the visibility of the fields, on top of that, why aren't you inlining this directly? Why a static variable?